app/messages.py

```diff
@@ -156,177 +147,141 @@
-    "message": "You cannot accept a"
-    " mentorship relation where you are"
-    " not involved."
-}
-USER_CANT_REJECT_REQUEST_SENT_BY_USER = {
-    "message": "You cannot reject a" " mentorship request sent by yourself."
+    "message": "You cannot accept a mentorship relation where you are not involved."
 }
+USER_CANT_REJECT_REQUEST_SENT_BY_USER = {"message": "You cannot reject a mentorship request sent by yourself."}
 CANT_REJECT_UNINVOLVED_RELATION_REQUEST = {
-    "message": "You cannot reject a"
-    " mentorship relation where you are"
-    " not involved."
-}
-CANT_CANCEL_UNINVOLVED_REQUEST = {
-    "message": "You cannot cancel a mentorship" " relation where you are not involved."
-}
-CANT_DELETE_UNINVOLVED_REQUEST = {
-    "message": "You cannot delete a mentorship" " request that you did not create."
-}
-NOT_IN_MENTORED_RELATION_CURRENTLY = {
-    "message": "You are not in a current" " mentorship relation."
-}
-USER_IS_INVOLVED_IN_A_MENTORSHIP_RELATION = {
-    "message": "You are currently" " involved in a" " mentorship relation."
-}
-USER_NOT_INVOLVED_IN_THIS_MENTOR_RELATION = {
-    "message": "You are not involved" " in this mentorship relation."
-}
-USER_USES_A_USERNAME_THAT_ALREADY_EXISTS = {
-    "message": "A user with that" " username already exists."
-}
-USER_USES_AN_EMAIL_ID_THAT_ALREADY_EXISTS = {
-    "message": "A user with that " "email already exists."
-}
-USER_IS_NOT_REGISTERED_IN_THE_SYSTEM = {
-    "message": "You are not registered in" "the system."
-}
+    "message": "You cannot reject a mentorship relation where you are not involved."
+}
+CANT_CANCEL_UNINVOLVED_REQUEST = {"message": "You cannot cancel a mentorship relation where you are not involved."}
+CANT_DELETE_UNINVOLVED_REQUEST = {"message": "You cannot delete a mentorship request that you did not create."}
+NOT_IN_MENTORED_RELATION_CURRENTLY = {"message": "You are not in a current mentorship relation."}
+USER_IS_INVOLVED_IN_A_MENTORSHIP_RELATION = {"message": "You are currently involved in a mentorship relation."}
+USER_NOT_INVOLVED_IN_THIS_MENTOR_RELATION = {"message": "You are not involved in this mentorship relation."}
+USER_USES_A_USERNAME_THAT_ALREADY_EXISTS = {"message": "A user with that username already exists."}
+USER_USES_AN_EMAIL_ID_THAT_ALREADY_EXISTS = {"message": "A user with that email already exists."}
+USER_IS_NOT_REGISTERED_IN_THE_SYSTEM = {"message": "You are not registered inthe system."}
 NAME_LENGTH_GREATER_THAN_MAX_LIMIT = {
-    "message": "The {field_name} field has"
-    " to be no more than {max_limit}"
-    " characters."
-}
-NAME_LENGTH_LESSER_THAN_MAX_LIMIT = {
-    "message": "The {field_name} field has to"
-    " be at least than {min_limit} characters"
-    " and no more than {max_limit}"
-    " characters."
-}
+    "message": "The {field_name} field has to be no more than {max_limit} characters."}
+NAME_LENGTH_LESSER_THAN_MAX_LIMIT = {"message": "The {field_name} field has to be at least than {min_limit} characters"
```

Comment:
I think this message description doesn't sounds good. It should be `"The {field_name} field has to be lesser than {min_limit} characters and not more than {max_limit} characters."`